libkbfs/folder_update_prepper.go

```diff
@@ -604,10 +604,14 @@ func (fup *folderUpdatePrepper) updateResolutionUsageLockedCache(
 	// we don't know whether these are files or directories, just look
 	// them up generically.  Ignore any recoverable errors for unrefs.
 	// Note that we can't combine these with the above ref fetches
-	// since they require a different MD.
+	// since they require a different MD.  If the merged changes
+	// didn't changes any blocks (in particular, the root block), we
+	// can assume all the blocks we are unreferencing were live;
```

Comment:
> I was gonna ask what if another client has changed the blocks resulting in block status change, but it seems we'd have been or will be rebased to the latest MD so that should never be a problem. Does it sound right?

Yeah exactly.  If there was a change in status that we don't picj up, it will result in an MD conflict, and we'll have to run CR again later.